backend/src/modules/task/dto/req.task-filter.dto.ts

```diff
@@ -0,0 +1,15 @@
+import { IsOptional, IsString } from 'class-validator';
+
+export class TaskFilterDto {
+  @IsOptional()
+  @IsString()
+  status?: string;
```

Comment:
please make sure its not just any string, it has to be the enum to be saved in db